Lock/DatabaseInteractor.swift

```diff
@@ -172,11 +168,7 @@ struct DatabaseInteractor: DatabaseAuthenticatable, DatabaseUserCreator, Loggabl
                     self.dispatcher.dispatch(result: .error(DatabaseUserCreatorError.passwordAlreadyUsed))
                 case .failure(let cause as AuthenticationError) where cause.isVerificationRequired:
                     Queue.main.async {
-                        self.webAuth.start(self.connection.name, loginHint: email, screenHint: "signup") { webAuthError in
-                            guard let webAuthError = webAuthError else { return callback(nil, nil) }
-                            self.logger.error("Failed web-based verification of user <\(email)> with error \(webAuthError)")
-                            callback(nil, .verificationFailure(error: webAuthError))
-                        }
+                        self.startWebAuth(loginHint: email, screenHint: "signup", callback: { callback(nil, $0) })
```

Comment:
I forgot to mention that the current bot detection flow only triggers for current users during a login attempt and never during the sign up flow.

Reply:
I'm seeing the bot detection response returned from Auth0 during signup, and the code already exists to handle that scenario
![signup](https://user-images.githubusercontent.com/915431/105572012-56f64100-5d09-11eb-893a-0e43dd2a17e8.gif)

Reply:
🤔 I will also bring this gif to the meeting. In our last Stephanie told Van this didn't happen